airtime_mvc/application/views/scripts/index/index.phtml

```diff
@@ -355,4 +355,14 @@
 
     });
 
-</script>
+</script><link href="/css/radio-page/custom.css" rel="stylesheet" type="text/css" />
```

Comment:
Did you mean to add this? Considering it is Google Analytics, please remove it from the PR

Reply:
Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.